### ISSUE TITLE
Make SEVIRI calibration more transparent to the user

### ISSUE THREAD

**Issue by sfinkens** [pytroll/satpy#2569]
## Feature Request

**Is your feature request related to a problem? Please describe.**
In some situations SEVIRI users request GSICS calibration but get Nominal calibration. Reason: If GISCS coefficients are not available for a certain channel, the SEVIRI calibration handler silently falls back to nominal coefficients. For users it's hard to know which coefficient has been applied. They would have to compare both calibration methods or check the header contents.

**Describe the solution you'd like**
- Add dataset attributes `calib_method` describing the calibration method and `calib_coefs` specifying the applied coefficients
- Update documentation to describe this behaviour
- Log a warning when falling back to Nominal coefficients

**Describe any changes to existing user workflow**
None

**Additional context**
None

**Comment by pdebuyl** [comment]
Question: if the user requests a specific set of calibration, shouldn't satpy raise an actual error?

**Comment by mraspaud** [comment]
> Question: if the user requests a specific set of calibration, shouldn't satpy raise an actual error?

I agree actually...

**Comment by sfinkens** [comment]
Good point! Then we would need an additional reader kwarg to allow mixing of coefficients.
